sdk/python/feast/loaders/ingest.py

```diff
@@ -120,8 +123,15 @@ def get_feature_row_chunks(
             Iterable list of byte encoded FeatureRow(s).
     """
 
+    def normalize_fields(fields):
+        return {field.name: field.dtype for field in fields.values()}
+
+    feature_set = f"{fs.project}/{fs.name}:{fs.version}"
+
+    normalized_fields = normalize_fields(fs.fields)
```

Comment:
Making `normalized_fields()` into a function seems unnecessary here as its only used once and outside of a `map()` or `filter()`.

Reply:
This was in response to https://github.com/gojek/feast/pull/536#discussion_r413949092 which in turn followed from https://github.com/gojek/feast/pull/536#discussion_r399608820

I'd still say if the author thinks giving it a name helps clarity, it's a stylistic choice regardless of whether it's reused or not. But if it's a poor name, that's another matter 😅

Reply:
As Ches mentioned, I originally followed the author's intention, but now changed. Thinking from perspective of the code reader - it should be clear enough what this code is doing by reading the variable name.